djangocms_snippet/models.py

```diff
@@ -106,6 +106,10 @@ class SnippetPtr(CMSPlugin):
 
     search_fields = ['snippet__html'] if SEARCH_ENABLED else []
 
+    def get_short_description(self):
+        snippet_label = SnippetGrouper.objects.filter(pk=self.snippet_grouper.pk).order_by('pk').first()
```

Comment:
```suggestion
        snippet_label = SnippetGrouper.objects.filter(pk=self.snippet_grouper.pk).first()
```
Unlike contents we will never have more than one grouper for a pk so ordering is a wasted db task, the filter will protect nothing being returned so that is ok to stay.